src/Microsoft.DotNet.Darc/src/Darc/Options/AddBuildToChannelCommandLineOptions.cs

```diff
@@ -59,7 +59,7 @@ internal class AddBuildToChannelCommandLineOptions : CommandLineOptions
         [RedactFromLogging]
         public string ArtifactPublishingAdditionalParameters { get; set; }
 
-        [Option("publish-installers-and-checksums", HelpText = "Whether installers and checksums should be published.")]
+        [Option("publish-installers-and-checksums", HelpText = "Whether installers and checksums should be published. This option is ignored")]
```

Comment:
Should we be thinking about getting rid of this?

Reply:
Yes, but I am pretty sure there are places passing it right now which will break if I remove the option.

Reply:
Yep, definitely not something for this PR.